components/capi/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "icu_capi"
+version = "0.1.0"
+authors = ["Manish Goregaokar <manishsmail@gmail.com>"]
```

Comment:
Use the "The ICU4X Authors" boilerplate

Reply:
I wonder if `.cargo` should be given a `default-author` key